model/generate.go

```diff
@@ -312,6 +299,15 @@ func (g *GeneratedProject) saveNewBuildsAndTasks(ctx context.Context, v *Version
 	return nil
 }
 
+// SetNewTasksAndActivationInfo sets the NewTVPairs and ActivationInfo fields on the generator.
+func (g *GeneratedProject) SetNewTasksAndActivationInfo(ctx context.Context, v *Version, p *Project) {
+	// Only consider batchtime for mainline builds. We should always respect activate if it is set.
```

Comment:
I know this was copy pasted from the original, but is this comment supposed to be here? Seems kind of a random place to put it. Is it supposed to be inside `findTasksAndVariantsWithSpecificActivations`?

Reply:
Yeah it initially prepended the function call so I kept it that way but true it makes more sense to move into `findTasksAndVariantsWithSpecificActivations `